cfe_internal/update/update_policy.cf

```diff
@@ -308,10 +312,33 @@ bundle agent cfe_internal_update_policy_cpv
         usebundle => cfe_internal_setup_python_symlink("$(python_symlink)");
 }
 
+bundle agent cfe_internal_update_cmdb
+# @brief Ensure local cache of CMDB data is up to date
+{
+  files:
+      "$(sys.workdir)/data/." create => "true";
+
+      "$(sys.workdir)/data/host_specific.json" -> { "ENT-6788" }
+        copy_from => u_cmdb_data;
+}
 #########################################################
 # Self-contained bodies from the lib to avoid dependencies
 #########################################################
 
+body copy_from u_cmdb_data
+# @brief Sync CMDB data from policy server
+#
+# Note: Not all hosts necessarily have CMDB data
+{
+        copy_backup => "false";
+        trustkey => "false";
+        compare => "digest";
+        source  => "/var/cfengine/cmdb/$(sys.key_digest)/host_specific.json";
```

Comment:
Do we have a variable for "sys.workdir on the policy server" ?

Reply:
No, not that I know of.

Reply:
I don't want to try and get it from `remotescalar()` either (Enterprise only, plus kind of painful to use).

I could however look to leverage a `shortcut` letting the server decide where this file actually is.